style.css

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
![image](https://user-images.githubusercontent.com/104353194/168750019-57d83138-2ec6-44c8-b963-c5703ffba4f6.png)
uppercase and sans-serif